services/payments/errors.go

```diff
@@ -28,3 +28,15 @@ type InsufficientAuthorizationsError struct{}
 func (e *InsufficientAuthorizationsError) Error() string {
 	return "insufficient authorizations"
 }
+
+const (
+	SolanaTransactionUnknownError Error = "transaction status unknown"
+	SolanaTransactionNotFoundError Error = "transaction not found"
+	SolanaTransactionNotConfirmedError Error = "transaction not confirmed"
```

Comment:
I'm sure there was a glitch in the Matrix, and you forgot that errors values in Go conventionally start with `Err`:

```suggestion
	ErrSolanaTransactionUnknown Error = "transaction status unknown"
	ErrSolanaTransactionNotFound Error = "transaction not found"
	ErrSolanaTransactionNotConfirmed Error = "transaction not confirmed"
```